public/about-team.html

```diff
@@ -279,91 +269,86 @@ <h2>Mechanical</h2>
             <div class="team-container">
               <div class="team-lead-card" id="6010a0ec96175608a4aaba37">
                 <div class="team-lead-img">
-                  <img src="images/teamleads/IfranM.jpg" />
+                  <img src="images/teamleads/placeholder.jpg" />
                 </div>
                 <div class="team-lead-name">Body Lead</div>
-                <div class="team-lead-text">Irfan Muzhaffar</div>
-                <div class="team-lead-text">Mechanical Engineering</div>
-                <div class="team-lead-text">Junior</div>
+                <div class="team-lead-text">Zachary Anderson</div>
               </div>
               <div class="team-lead-card" id="5c227d8b0570397fe37636ec">
                 <div class="team-lead-img">
-                  <img src="images/teamleads/ZachA.jpg" />
+                  <img src="images/teamleads/placeholder.jpg" />
                 </div>
                 <div class="team-lead-name">Braking Lead</div>
-                <div class="team-lead-text">Zachary Anderson</div>
-                <div class="team-lead-text">Mechanical Engineering</div>
-                <div class="team-lead-text">Junior</div>
+                <div class="team-lead-text">Kyle Ma</div>
               </div>
               <div class="team-lead-card" id="5d81361623ef706228beb7e8">
                 <div class="team-lead-img">
-                  <img src="images/teamleads/BrookeE.jpg" />
+                  <img src="images/teamleads/placeholder.jpg" />
```

Comment:
Brooke was the chassis lead before she went on co op - we should have her picture